### ISSUE TITLE
home directory assumed rather than read

### ISSUE THREAD

**Issue by GoogleCodeExporter** [jimradford/superputty#331]
```
What steps will reproduce the problem?
1. open scp session with non standard home (ex root -- home is /root not 
/home/root)
2. connection opens but unable to navigate because /home/username does not exist
3.

What is the expected output? What do you see instead?
home is read from remote server and used for scp

What version of the product are you using? On what operating system?
1.4.0.4 windows server 2008 r2

Please provide any additional information below.
```

Original issue reported on code.google.com by `ada...@scheblein.net` on 28 May 2013 at 3:47

**Comment by jimradford** [comment]
This is fixed in 1.4.0.6 with a configuration setting change in Options (F2)

**Comment by amarbit** [comment]
I am getting error and not able to see current path files: Error is like parsing directory error
